QUICK_START.md

```diff
@@ -0,0 +1,153 @@
+_MIT License_
+_Copyright (c) 2017 MassChallenge, Inc._
+
+## Quickstart for Impact-API
+
+1. Install [Docker](https://docs.docker.com/docker-for-mac/)
+(download the stable channel).
+
+2. Start Docker from the Applications folder.
+
+3. Get the source code.  If you already have impact-api checked out,
+then in a Terminal window go to that directory and bring it up to date
+with:
+```
+cd <impact-api-directory>
+git checkout development
+git pull
+```
+
+If you don't have impact-api checked out, but you do have the
+accelerate code base checked out, checkout the impact-api code
+next to the accelerate directory with:
+```
+cd <accelerate-api-directory>/..
+git clone https://github.com/masschallenge/impact-api.git
+cd impact-api
+```
+
+If you don't have the accelerate code base checked out, you will
+need it to get an initial database.  If you need this, then contact
+engineering@masschallenge.org.
+
+4. Run tests.  From the impact-api source directoy run:
+```
+make build
+make test
+```
+
+This will take a while, but all tests should pass.  Specifically the
+end of the output should look something like:
+
+```
+Ran 102 tests in 14.845s
+
+OK
+Destroying test database for alias 'default'...
+```
+
+4. Start server
+
+```
+make dev
+```
+
+Once the process completes (it will take a little while and the last
+line should sya "BUILD COMPLETE"), visit http://localhost:8000 in a
+browser to see the login screen.
+
+5. In a separate shell, load an accelerate database.  Assuming
+you have an up-to-date accelerate source code directory next to
```

Comment:
The reference to "again" is unclear since I'm not seeing any other comments.  These "ifs" were in large part added when reviewing this with Jeff so that he could get his environment working.  The document you imply would be interesting, but does not achieve the "done" of the ticket.  Won't fix.